components/ingest-service/backend/elastic/mappings/converge.go

```diff
@@ -3,326 +3,323 @@ package mappings
 // ConvergeHistory is the representation of our `converge-history` Mapping
 var ConvergeHistory = Mapping{
 	Index:      "converge-history",
-	Type:       "converge",
 	Timeseries: true,
 	Mapping: `
 	{
-		"template": "converge-history-*",
```

Comment:
if we are removing the template, in some tests we are trying to delete the templates.
So with these, I hope we can remove that test as well. 
https://github.com/chef/automate/blob/main/components/compliance-service/Makefile#L280

Reply:
not needed in opensearch, will throw error if we keep.